### PR TITLE
Adding bridge interface module for Cumulus Linux

### PR DESCRIPTION
**Pull request by LeslieCarr**

### CODE REVIEW

File: network/cumulus/cl_bridge.py [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+        description:
+            - set spanning tree root priority. Must be a multiple of 4096
+    location:
+        description:
+            - interface directory location
+        default:
+            - /etc/network/interfaces.d
+
+
+requirements: [ Alternate Debian network interface manager - \
```
Comment (bcoca) [inline network/cumulus/cl_bridge.py]: this breaks documentation parsing

**Comment by robynbergeron** [comment]
Hi @LeslieCarr -- thanks for submitting this new module.

I noticed that the author line has an email address but not a github ID associated with it -- 
can you update that, so it adheres to the guidelines? (http://docs.ansible.com/developing_modules.html#module-checklist)

When you're done with that (and/or with addressing @bcoca's note on documentation parsing breakage) please add a comment with the text "ready_for_review" and we will continue with the review process.

Thanks! :D

**Comment by LeslieCarr** [comment]
ready_for_review

**Comment by robynbergeron** [comment]
Hi @LeslieCarr -- Adding info on new PR process. We will be evaluating all new module PRs according to this process, effective immediately. 

# 

Thanks for submitting this new module to Ansible Extras! This module is now in community review, a process that is open to all Ansible users. In order for this module to be approved, it must gain the following votes:

“works_for_me”: If you have tested the module thoroughly, including testing of all of the module’s options, and if the module works for you, please add “works_for_me” in the comments.

“passes_guidelines”: If you have gone through the module guidelines and the module meets all of the requirements, please add “passes_guidelines” in the comments. Guidelines are available here: http://docs.ansible.com/developing_modules.html#module-checklist

“needs_revision”: If the module fails to work for you, or if it doesn’t meet guidelines, please add “needs_revision” in the comments with details about what needs to be fixed.

When a module has both “works_for_me” and “passes_guidelines” tags, we will promote the module for inclusion in Ansible Extras.  At this point, you will be expected to maintain the module by fixing bugs and evaluating pull requests in a timely manner. 

Thanks again for submitting your Ansible module!

**Comment by privateip** [comment]
works_for_me

**Comment by robynbergeron** [comment]
@privateip -- thanks for the functionality vote :)

I don't suppose you happened to also notice if it passed the module guidelines as well? :D If not, we'll wait on a vote for that as well.

**Comment by privateip** [comment]
doh!  will go back through them tonight and retag as needed

**Comment by privateip** [comment]
needs_revision
- please add version_added: "2.1" to the DOCUMENTATION string
- please add RETURN (see http://docs.ansible.com/ansible/developing_modules.html#documenting-your-module)

**Comment by gregdek** [comment]
Thanks @LeslieCarr for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

**Comment by privateip** [comment]
Closed per discussions with author

**Pull request closed without merging**